packages/transformers/js/src/visitors/fs.js

```diff
@@ -61,10 +61,10 @@ export default {
       } else {
         replacementNode = t.stringLiteral(res);
       }
-
-      asset.connectedFiles.push({
-        filePath: filename
-      });
+      console.log('ADDING CONNECTED FILE', file);
+      // asset.connectedFiles.push({
```

Comment:
This was screwing up the hashes because it was adding the `package.json` many times, which was part of the hash on cache write, but not on cache read where it was deduplicated once put in the graph. Decided to comment out for now because this should probably be brought in with a config request in the future anyway.